src/anemoi/training/data/datamodule.py

```diff
@@ -125,6 +125,8 @@ def ds_valid(self) -> NativeGridDataset:
         r = self.rollout
         if self.config.diagnostics.eval.enabled:
             r = max(r, self.config.diagnostics.eval.rollout)
+        if self.config.diagnostics.plot.longrollout.enabled:
```

Comment:
If both if-statements are enabled, the latter will overwrite the value r, is that intentional?